storage/rocksdb/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ STRING(REGEX REPLACE "-Wframe-larger-than=[0-9]*" ""
        CMAKE_CXX_FLAGS_RELWITHDEBINFO "${CMAKE_CXX_FLAGS_RELWITHDEBINFO}")
 STRING(REGEX REPLACE "-Wframe-larger-than=[0-9]*" ""
        CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG}")
-MY_CHECK_AND_SET_COMPILER_FLAG(-Wframe-larger-than=32768)
+MY_CHECK_AND_SET_COMPILER_FLAG(-Wframe-larger-than=49152)
```

Comment:
I’m okay with this change, but it’s better with a comment from RocksDB about why they add this replacement in the first place.

Reply:
https://github.com/MariaDB/server/commit/82d7419e0600a70b1a1c993d33ed6cf79fbd6129#diff-2acd82b249609d071929a6a0d083e36ef5b8807294bfe610e6552211c0c2b30e

So we added the first frame size. clang for some reason was a higher requirements.

The error was deep in a library in their submodule so not easily possible to work around.

Reply:
some dirty source directory manifested a wild goose chase where an unrecognisable frame side from a crc32c_3way function occured.

Cleaned it out and never saw it gain.